ironmon_tracker/Theme.lua

```diff
@@ -618,18 +609,18 @@ function Theme.openSaveCurrentThemeWindow()
 		-- Check a few conditions that would prevent the user from using a particular Theme name
 		if existingPresetIndex == Theme.PresetsIndex.ACTIVE or existingPresetIndex == Theme.PresetsIndex.DEFAULT then
 			-- Don't allow importing "Active Theme (Custom)" or "Default Theme" as that is reserved
-			forms.settext(Theme.Manager.SaveNewWarning, "Cannot use a reserved Theme name")
-			forms.settext(Theme.Manager.SaveNewConfirm, Resources.AllScreens.Save)
+			ExternalUI.BizForms.setText(Theme.Manager.SaveNewWarning, "Cannot use a reserved Theme name")
+			ExternalUI.BizForms.setText(Theme.Manager.SaveNewConfirm, Resources.AllScreens.Save)
 			return
 		elseif themeName:find("%x%x%x%x%x%x") then
 			-- Don't allow six consectuive hexcode characters, as this screws with parsing it later
-			forms.settext(Theme.Manager.SaveNewWarning, "Name cannot have 6 consectuive hexcode characters (0-9A-F)")
-			forms.settext(Theme.Manager.SaveNewConfirm, Resources.AllScreens.Save)
+			ExternalUI.BizForms.setText(Theme.Manager.SaveNewWarning, "Name cannot have 6 consectuive hexcode characters (0-9A-F)")
+			ExternalUI.BizForms.setText(Theme.Manager.SaveNewConfirm, Resources.AllScreens.Save)
 			return
-		elseif existingPreset ~= nil and forms.gettext(Theme.Manager.SaveNewConfirm) ~= Resources.AllScreens.Yes then
+		elseif existingPreset ~= nil and ExternalUI.BizForms.getText(Theme.Manager.SaveNewConfirm) ~= Resources.AllScreens.Yes then
 			-- If the Theme name is already in use, warn the user first
-			forms.settext(Theme.Manager.SaveNewWarning, "A Theme with that name already exists. Overwrite?")
-			forms.settext(Theme.Manager.SaveNewConfirm, Resources.AllScreens.Yes)
+			ExternalUI.BizForms.setText(Theme.Manager.SaveNewWarning, "A Theme with that name already exists. Overwrite?")
```

Comment:
should these warning texts be moved to resources language files to be translatable?

Reply:
Technically yes, but I did not put all texts in the Resource file for translation help, mostly because it was just so many extra lines of texts to add. I wanted to just add the core texts to Resource files.

If/when this gets fixed, then it can be done in `dev` branch